apps/web/app/(navigation)/budgets/[id]/TransactionsTable.tsx

```diff
@@ -201,49 +180,46 @@ export const TransactionsTable = ({
               switch (props.column.key) {
                 case "date":
                   const value = props.groupKey[props.groupIndex];
-                  return <>{getDayName(value)}</>;
+                  return <>{getLocaleDateString(value)}</>;
```

Comment:
@wlechowicz can we just return a string here instead of a fragment, or does it not matter at all?
```suggestion
                  return getLocaleDateString(value);
```